### PR TITLE
Eg style nav

### PR DESCRIPTION
**Pull request by erikgrueter**

### CODE REVIEW

**Comment by dreamingfurther** [comment]
Why the deleted feature test? Also it looks like you should merge master into the branch and re-push.

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -13,14 +13,18 @@
...
  *= require_self
  *= require foundation_and_overrides
- *= require font-awesome
- */
 
-@import "modules/landing";
+ */
 
+<<<<<<< HEAD
+@import 'modules/topbar';
```
Comment (erikgrueter) [inline app/assets/stylesheets/application.css.scss]: Not sure why this is incorrect!

File: app/assets/stylesheets/modules/footer.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+    height: 100x;
+    overflow: hidden;
+    position: relative;
+    width: 100px;
+
+    img {
```
Comment (erikgrueter) [inline app/assets/stylesheets/modules/footer.scss]: Is nesting not the point of sass?

Comment (dreamingfurther) [inline app/assets/stylesheets/modules/footer.scss]: there is a difference in nesting vs depth of applicability. and deep nesting is actually a css antipatern.

**Comment by dreamingfurther** [comment]
@erikgrueter 
it looks like this broke the travis build, can you fix before re-pushing? looks like it also needs to have master merged into it

**Comment by dreamingfurther** [comment]
good work! feel free to figure out the merge and push this into master @erikgrueter

**Pull request merged**